src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -573,6 +582,31 @@ public function getCMSFields()
         return $fields->makeReadonly();
     }
 
+    /**
+     * Generate a Dropdown field with the list of possible job
+     */
+    private function buildJobStatusField(): DropdownField
+    {
+        $statuses = $this->getJobStatusValues();
+        return DropdownField::create(
+            'JobStatus',
+            $this->fieldLabel('JobStatus'),
```

Comment:
```suggestion
            _t(__CLASS__ . '.TABLE_STATUS', 'Status'),
```
This is what's used in `summaryFields()` so it will keep localisations consistent.